src/AutomatedCar/SystemComponents/GearShifter.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace AutomatedCar.SystemComponents
+{
+    // A 4 alap valtoallas, amit inputkent kapunk a HMI-tol
+    public enum GearShifterPosition
```

Comment:
Miért egy ugyanilyen enum két helyen is?

Reply:
lemaradt egy push, a masik helyrol ki lett veve - elmeletileg mar javitottam es latszik is